Makefile

```diff
@@ -22,7 +22,7 @@ build:
 	mv textile dist/
 
 install:
-	mv dist/textile /usr/local/bin
+	mv dist/textile $$GOPATH/bin
```

Comment:
Saw the convo around this. What is the implication of this? GOPATH is usually added to default search PATH right? So still available via cli easily?

Reply:
Best practice says to add it to your `PATH` via bash profile, etc...

Reply:
Yeh, cool, nice.